aFWall/src/main/java/dev/ukanth/ufirewall/MainActivity.java

```diff
@@ -487,6 +488,12 @@ private boolean passCheck() {
                     } else {
                         requestPassword();
                     }
+                case "p3":
```

Comment:
I guess we might need break statement, since the existing "p2" option doesn't have break;